bayesian/auth.py

```diff
@@ -15,6 +15,23 @@
 
 jwt.register_algorithm('RS256', RSAAlgorithm(RSAAlgorithm.SHA256))
 
+def decode_token():
```

Comment:
@fridex need more inputs perhaps. AFAIK, decode_token now includes the entire decoding token logic and the login_required wrapper does not duplicate any of that. As you can see, login_required uses decode_token.

The current design of the decode_token allows it to be used in other places like stack-analyses and component-analyses logic in api_v1.py.